src/main/java/mainwindow/MainWindow.java

```diff
@@ -0,0 +1,80 @@
+package mainwindow;
+
+import javafx.animation.PauseTransition;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.util.Duration;
+import mollyexception.MollyException;
+import ui.DialogBox;
+import ui.Ui;
```

Comment:
The import order does not follow the style guide. Perhaps you've forgotten to reformat the file or have not configured IntelliJ? I noticed the same issue throughout the codebase.

https://se-education.org/guides/tutorials/intellijCodeStyle.html